cookie-banner-rules-list.json

```diff
@@ -636,6 +636,265 @@
+  },
+  "domain": "msn.com",
+  "schema": 1665665451546,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "eupubconsent-v2",
+        "value": "CPg0HlKPg0HlKAcABBENCFCsAP_AAEPAACiQImtf_X__b3_j-_5_f_t0eY1P9_7_v-0zjhedt-8N3d_X_L8X52M7vF36pq4KuR4Eu3LBAQdlHOHcTUmw6IkVqzPsbk2cr7NKJ7PEmnMbO2dYGH9_n93TuZKY7______z_v-v_v____f__-3_3__5_1---wAAB8gdzLv9____393P___9v-_9_____-CIYBJhqXkAXZljgybRpVCiBGFYSHQCgAooBhaIrCB1cFOyuAn1BCwAQCpCcCIEGIKMGAQACCQBIREBIAeCARAEQCAAEAKsBCAAjYBBYAWBgEAAoBoWIEUAQgSEGRwVHKYEBUi0UE9lYglB3saYQhlnARQIAAABABrNECwMhIWDmOAJAS8WSB5ihfIBAaBCAFYALgAhgBkADLAGyAOwAfgBAACCgEYAKeAVeAtAC0gGsAN4AdUA-QCHQEVAJEATYAnYBSIC5AGEgMPAYwAycBnIDPAGfAPwDACgBzAHUASEAkUBkYDdAHEgOzAe6BD4QALABIAOYA3gCQgEigN0AcSA7MB7oD7AIfCIDQAVgBDADIAGWANkAdgA_ACAAEYAKeAVcA1gB1QD5AIdASIAmwBOwCkQFyAMJAYeAycBnIDPgH4CoDgAFAAhgBMAC4AI4AZYA7AB-AEYAI4AVeAtAC0gG8ASCAmIBNgCmwFsALkAXmAw8BkQDOQGeAM-AbkA_ACF4oAaANoAcwA8ACCgHVAR6AkUBrwDbwHEgPsAgeBBsZAaAAoAEMAJgAjgBlgDsgH2AfgBGACOAFXAK2AbwBJwCYgE2ALRAWwAvMBh4DIgGcgM8AZ8A-IB-AELxgAwAbQA5gB4AFiAOqAj0BIoC8gG3gOJAfYBBsAA.f_gACHgAAAAA"
+      }
+    ],
+    "optOut": [
+      {
+        "name": "eupubconsent-v2",
+        "value": "CPg0HVBPg0HVBAcABBENCFCgAAAAAAAAACiQAAAAAAEBoEIAVgAuACGAGQAMsAbIA7AB-AEAAIKARgAp4BV4C0ALSAawA3gB1QD5AIdARUAkQBNgCdgFIgLkAYSAw8BjADJwGcgM8AZ8A_AMAKAHMAdQBIQCRQGRgN0AcSA7MB7oEPhAAsAEgA5gDeAJCASKA3QBxIDswHugPsAh8IgNABWAEMAMgAZYA2QB2AD8AIAARgAp4BVwDWAHVAPkAh0BIgCbAE7AKRAXIAwkBh4DJwGcgM-AfgKgOAAUACGAEwALgAjgBlgDsAH4ARgAjgBV4C0ALSAbwBIICYgE2AKbAWwAuQBeYDDwGRAM5AZ4Az4BuQD8AIXigBoA2gBzADwAIKAdUBHoCRQGvANvAcSA-wCB4EGxkBoACgAQwAmACOAGWAOyAfYB-AEYAI4AVcArYBvAEnAJiATYAtEBbAC8wGHgMiAZyAzwBnwD4gH4AQvGADABtADmAHgAWIA6oCPQEigLyAbeA4kB9gEGwAAA.YAAAAAAAAAAA"
+      }
+    ]
+  },
+  "id": "dbbf625a-5a3f-4040-96c3-4633cc2d3e24"
+  },
+    {
+  "click": {
+    "optIn": "button.primary",
+    "presence": "div.con-wizard"
+  },
+  "domain": "yahoo.com",
+  "schema": 1665666115860,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "EuConsent",
+        "value": "CPgyVIAPgyVIAAOACBROCkCoAP_AAH_AACiQJBNd_H__bX9n-f7_6ft0eY1f9_r37uQzDhfNk-8F3L_W_LwX_2E7NF36tq4KmR4ku1LBIUNtHMnUDUmxaokVrzHsak2cpzNKJ_BkknsZe2dYGF9vm5tj-QKZ7_5_d3f52T_9_9v-39z33913v3d93-_13LjdV5_9H_v9fR_bc_Kft_5-_8v8_____3_e______8EggCTDVuIAuxLHAm0DCKBECMKwkKoFABBQDC0QGADg4KdlYBPrCBAAgFAEYEQIcAUYEAgAAAgCQiACQIsEAAAIgEAAIAEQiEABAwCCgAsDAIAAQDQMUQoABAkIMiAiKUwICIEggJbKhBKC6Q0wgDrLACgkRsVAAiAAAUgACAsHAMESAlYsECTFG-QAjBCgFEqFaAGGgAwABBIIRABgACCQQqADAAEEgg",
+        "isSecure": true
+      }
+    ]
+  },
+  "id": "2f9123b1-7d6b-460f-8da8-988f82238ec7"
+  },
+    {
+  "click": {
+    "optIn": "button.cmd-button-accept-all",
+    "optOut": "button.cmd-button-reject-all",
+    "presence": "div.cookies-modal"
+  },
+  "domain": "xhamster.com",
+  "schema": 1665666942133,
+  "cookies": {
+    "optIn": [
+      {
+        "name": "cookie_accept",
+        "value": "%7B%22e%22%3A1%2C%22n%22%3A1%2C%22tp%22%3A1%7D"
+      },
+      {
+        "name": "parental-control",
```

Comment:
Is this choice really cookie banner related? I want to make sure we're not hiding any UI other than cookie banners.

Reply:
I've retested this one and it appears only after interacting with the cookie banner. We can remove it if we consider it unnecessary.

Reply:
Removed the unnecessary cookie.